src/Groups/group_characters.jl

```diff
@@ -612,6 +646,32 @@ function(chi::GAPGroupClassFunction)(g::BasicGAPGroupElem)
     error("$g is not an element in the underlying group")
 end
 
+function class_positions_of_kernel(chi::GAPGroupClassFunction)
+    deg = degree(chi)
+    return filter(i -> chi[i] == deg, 1:length(chi))
+end
+
+function class_positions_of_kernel(list::Vector{T}) where T
+    length(list) == 0 && return T[]
+    deg = list[1]
+    return filter(i -> list[i] == deg, 1:length(list))
+end
+
+@doc Markdown.doc"""
+    indicator(chi::GAPGroupClassFunction, n::Int = 2)
+
+Return the `n`-th Frobenius-Schur indicator of `chi`, that is,
+the value $(∑_{g ∈ G} chi(g^n))/|G|$, where $G$ is the group of `chi`.
+
+If `chi` is irreducible then `indicator(chi)` is
+`0` if `chi` is not real-valued,
+`1` if `chi` is afforded by a real representation of $G$, and
+`-1` if `chi` is real-valued but not afforded by a real representation of $G$.
+"""
+function indicator(chi::GAPGroupClassFunction, n::Int = 2)::Int
+    return GAP.Globals.Indicator(chi.table.GAPTable, GAP.GapObj([chi.values]), n)[1]
```

Comment:
```suggestion
function indicator(chi::GAPGroupClassFunction, n::Int = 2)
    return GAP.Globals.Indicator(chi.table.GAPTable, GAP.GapObj([chi.values]), n)[1]::Int
```
The code you have right now indicates that the return value shall be *converted* to `Int`, via `convert(Int, ...)`; while my suggestion is just a type assertion, which may be slightly cheaper.

Reply:
O.k.